source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -18,6 +18,8 @@
 #include <cstring>
 #include <memory>
 #include <string>
+#include <vector>
+#include <tuple>

```

Comment:
Sorry, missed this on the first pass, please keep them in alphabetical order as well. I wish we had a tool to do this automatically...